theano/sandbox/jaxify.py

```diff
@@ -984,3 +985,16 @@ def ravelmultiindex(*inp, mode=mode, order=order):
         return jnp.ravel_multi_index(multi_index, dims, mode=mode, order=order)
 
     return ravelmultiindex
+
+
+# TODO: Remove all these when done
+# https://jax.readthedocs.io/en/latest/_autosummary/jax.numpy.eye.html#jax.numpy.eye
+@jax_funcify.register(Eye)
+def jax_funcify_Eye(op):
+    # order = op.order
+
+    def eye(*inp, order="C"):
+        N, M, k = inp
```

Comment:
The signature can/should reflect the known arguments (i.e. `def eye(N, M, k, ...)`).

Reply:
Fixed I think?